app/src/main/java/nerd/tuxmobil/fahrplan/congress/designsystem/colors/ColorScheme.kt

```diff
@@ -0,0 +1,92 @@
+package nerd.tuxmobil.fahrplan.congress.designsystem.colors
+
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.material3.ColorScheme as Material3ColorScheme
+
+@Immutable
+data class ColorScheme(
```

Comment:
Should this one be named `EventColorScheme` similar to `EventFahrplanTheme` because this is the custom class?

Reply:
That makes it sound like it's a color scheme related to events specifically.

If you need a prefix, I'd go with `EventFahrplanColorScheme`. But sooner or later it'll be the only `ColorScheme` available to the app code. So I think it's fine to leave it like this.

Reply:
My reason to prefix came from the confusion between framework color scheme(s) and the app custom color scheme. I thought a prefix would make it easier to distinguish them.

Reply:
I think of it as wrapping and extending the functionality that the Material 3 library offers. It's similar to having a design system component called `Text` and the Material component called `Text`. There's only a name clash in the design system code/module. Ideally, the app code should only see the design system variant, and so there shouldn't be any confusion.

I'd keep the short name. But I can change it if you really want me to.